### DISCUSSION TITLE
Strong name assembly

### DISCUSSION THREAD

**Discussion by leandrogomes777** [jchristn/WatsonTcp#233] [Ideas]
Hi guys. 
Could you sign the assemblys? I need a strong name assembly in the project.

Thanks,

**Comment by jchristn** [comment]
Hi, sorry for the delay.  I hope you had a nice holiday.

NuGet: https://www.nuget.org/packages/WatsonTcp/5.1.0
Commit: https://github.com/jchristn/WatsonTcp/commit/561f4bf2c890e1fa49c27c210d75e91698761aff
